nntrainer/layers/acti_func.h

```diff
@@ -344,6 +347,27 @@ class ActiFunc {
                                     : static_cast<T>(NEGATIVE_SLOPE);
   }
 
+  /**
+   * @brief     Softplus activation function
+   * @tparam T type of an input/output
+   * @param x input
+   * @return T type output
+   */
+  template <typename T = float> static T softplus(T x) {
+    /** TODO: Change beta to be a property */
```

Comment:
For now, It is OK. but we need to have proper API for acti_function to have properties.